.env

```diff
@@ -0,0 +1,2 @@
+export APP_SETTINGS="config.DevelopmentConfig"
+export DATABASE_URL="postgresql://localhost/genten"
```

Comment:
Is there no way we could use something like sqlite3 or a similar local driver for now rather than having to install a postgres application?

Reply:
Updated, if we deploy as a production instance we have to set an ENV variable with the path to any supported DB. As long as we set the config to DevelopmentConfig it will use sqlite